script.js

```diff
@@ -1,12 +1,106 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let myImages = ['Winnie-the-Pooh.png', 'tinker-bell.png', 'simba.png', 'dumbo.png', 'Max-andRuby.png', 'disney-princesses.png']
+let winnie;
+let brushSize = 0;
+let brushColor = 0;
+let playButton;
+let song;
+let slider;
+let imageUse;
+
+function preload() {
+  slider = createSlider(5, 100, 20, 1);
+  slider.position(975, 130);
+  slider.style('width', '250px');
+
+  imageUse = loadImage("./assets/" + random(myImages));
+  playButton = loadImage("assets/play-button.png")
+  song = loadSound("./assets/Le-Festin-Camille");
+  
+
+}
 
 function setup() {
   createCanvas(windowWidth, windowHeight);
-    background(255);
+  frameRate(32);
+ 
+
+  textSize(24);
+  fill('red');
+  text('Hit Space in your key board to save image.', 825, 1000);
+
+  textSize(20);
+  fill(0);
+  text('Brush Size: ', 850, 137);
+
+  imageUse.resize(800, 1050);
+  image(imageUse, 0, 0);
+
+  playButton.resize(50, 50);
+  image(playButton, 1300, 800);
+  
 }
 
 function draw() {
+  
+  fill('red');
```

Comment:
could potentially turn this into a for loop, with a list of colors and some math to get the ellipse locations